### PR TITLE
Rating component Changes

### PR DESCRIPTION
**Pull request by nitheshkumar-hcl**
Comitting the changes for rating component story.

### CODE REVIEW

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    @Override
+    public JSONObject saveRating(JSONObject ratItem) {
+        boolean rowInserted = false;
+        JSONObject rateItemRes = new JSONObject();
+        Connection connection = getConnection();
+         try {
+
+            query = "INSERT INTO hclecomm.product_ratings(sku,name,rating) VALUES (?,?,?)";
+
+
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: @nitheshkumar-hcl  remove extra space

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+             catch (SQLException e) {
+                 log.error("Error while trying to close connection. SQLException = {}", e);
+             }
+         }
+        log.info("inserted data is {}",rowInserted);
+         return rateItemRes;
+    }
+
+
+
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: @nitheshkumar-hcl  remove extra space

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+        finally {
+            try {
+                if(null != connection && !connection.isClosed())
+                    connection.close();
+            }
+            catch (SQLException e) {
+                log.error("Error while trying to close connection. SQLException = {}", e);
+            }
+        }
+        log.info("Rating List is {}",ratingsList.toString());
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: @nitheshkumar-hcl  remove this or update to debug.

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+    }
+
+
+    private Connection getConnection() {
+
+        DataSource dataSource = null;
+        Connection con = null;
+        try{
+            log.info("GET CONNECTION");
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: @nitheshkumar-hcl  update to debug

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            dataSource = (DataSource) source.getDataSource("hclecomm");
+            con = dataSource.getConnection();
+            log.info("connection is returned");
```
Comment (rakeshkumar-verma) [inline core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: @nitheshkumar-hcl  update to debug logger.

Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: Updated to debug

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java (hunk 4 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: Updated to debug

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java (hunk 3 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: Updated to debug

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java (hunk 2 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: Removed

File: core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java (hunk 1 above)
Comment (nitheshkumar-hcl) [reply core/src/main/java/com/hcl/ecomm/core/services/impl/RatingServiceImpl.java]: Removed

**Pull request merged**